src/envo/envo-edit.owl

```diff
@@ -44005,6 +44006,14 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105126> "weathering of plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105126> <http://purl.obolibrary.org/obo/ENVO_06105119>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105212> (wind run)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/w/index.php?title=Wind_run&oldid=938497268") Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://www.weather.gov/media/coop/EVAP-OB_NovaLynx_ObserverGuide02June2015_ver1.pdf") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105212> "A wind measuring process which is determined by the total distance of the traveled wind over a period of time."@en)
```

Comment:
Thanks @KrishnaTO - this definition suggests that there's a "wind measuring process" superclass/genus - which I don't think is the case - could you explain the logic here?

Reply:
The intent was, to describe a specialized term (wind run), used to describe the quantity (over a period of time, however much wind distance the instrument managed to read), and hint at its unnamed process which uses an anememeter, time and good ol' maths.

Reply:
'wind run' is now under amount and the definition has been fixed to reflect that. I believe the PR can thus be merged